src/functionaltest/java/com/ericsson/ei/frontend/pageobjects/SubscriptionPage.java

```diff
@@ -181,53 +177,48 @@ public boolean expandButtonExist(String XPath) {
     }
 
     public boolean buttonExistByXPath(String XPath) {
-        // The row indicates weather or not the del / view and edit buttons has moved down to
+        // The row indicates weather or not the del / view and edit buttons has moved
+        // down to
```

Comment:
This is a really weird line break.

Reply:
Not my change. Had to rebase to latest code to solve the integrations test issues due to EI Backend changes. and got this into my PR.